packages/slate-react/src/utils/scroll-to-selection.js

```diff
@@ -80,6 +80,10 @@ function scrollToSelection(selection) {
     }
 
     selectionRect = range.getBoundingClientRect()
+
+    if (selectionRect.top == 0 && selectionRect.height == 0) {
```

Comment:
`if (selectionRect.top === 0 && selectionRect.height === 0)`

Reply:
as slate apparently uses improper `==` checks  throughout the code base... ignore this maybe?